### PR TITLE
BXMSDOC-5434-master: Update configuring business central settings and properties doc per changes in RHDM/RHPAM 7.7

### PR DESCRIPTION
**Pull request by kaldesai**
**JIRA:** 
- [Epic](https://issues.redhat.com/browse/BXMSDOC-5199)
- [Dedicated JIRA](https://issues.redhat.com/browse/BXMSDOC-5434)

**7.7 Doc previews:** 
- [RHPAM 7.7 Configuring Business Central settings and properties](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-5434-RHPAM-Maven-Archetype/#managing-business-central-listing-archetype-proc)
- [RHDM 7.7 Configuring Business Central settings and properties](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-5434-RHDM-Maven-Archetype/#managing-business-central-listing-archetype-proc)

@caponetto, Can please review and verify the doc content?
Added a new chapter as **CHAPTER 6. ARCHETYPE MANAGEMENT** in Configuring Business Central settings and properties doc.

### CODE REVIEW

File: doc-content/enterprise-only/admin-and-config/managing-business-central-listing-archetype-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+.Prerequisites
+
+* You have created an archetype and listed in the {CENTRAL} *Settings* from the maven repository.
+
+.Procedure
+
+. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Archetypes*.
++
+In the *Status* column, green icon indicates it is a valid archetype, red icon indicates it is an invalid archetype whereas blue icon indicates the corresponding archetype is the default one for new the spaces.
```
Comment (caponetto) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-listing-archetype-proc.adoc]:
```suggestion
In the *Status* column, green icon indicates it is a valid archetype, red icon indicates it is an invalid archetype whereas blue icon indicates the corresponding archetype is the default one for the new spaces. 
```

Comment (kaldesai) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-listing-archetype-proc.adoc]: Done

File: doc-content/enterprise-only/admin-and-config/managing-business-central-archetype-con.adoc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[id='managing-business-central-archetype-con']
+= Archetype management
+
+{CENTRAL} provides an archetype management features that enables you to list, add, validate, set as default, and delete the archetypes. You can manage archetypes from the *Archetypes* page in {CENTRAL}. Archetypes are projects that are installed in maven repositories in which a template structure is set or can be created if required.
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/admin-and-config/managing-business-central-archetype-con.adoc]: Done

File: doc-content/enterprise-only/admin-and-config/managing-business-central-adding-archetype-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[id='managing-business-central-adding-archetype-proc']
+= Adding an archetype
+
+You can add a new archetype to {CENTRAL}.
+
+.Prerequisites
+
+* You have installed an archetype in the Maven repository.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-adding-archetype-proc.adoc]: Prereq style: 

An archetype is installed in the Maven repository.

File: doc-content/enterprise-only/admin-and-config/managing-business-central-additional-features-archetype-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
+[id='managing-business-central-additional-features-archetype-proc']
+= Managing additional features of an archetype
+
+You can delete, set a default, and validate the archetypes from the *Archetypes* page in {CENTRAL}.
+
+.Prerequisites
+
+* You have created an archetype and listed in the {CENTRAL} *Settings* from the Maven repository.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/admin-and-config/managing-business-central-additional-features-archetype-proc.adoc]: You have created an archetype and "listed it"..

**Pull request merged**